www/js/diary/list.js

```diff
@@ -280,6 +278,38 @@ angular.module('emission.main.diary.list',['ui-leaflet',
 
     };
 
+    // Tour steps
+    var tour = window.tour = {
```

Comment:
why do you need to set `window.tour` to the tour? In general, angular discourages you from using window and encourages the use of `$scope` instead.

But I don't even see you using `window.tour` anywhere...